### ISSUE TITLE
About CLS model

### ISSUE THREAD

**Issue by csgogo** [AnTao97/dgcnn.pytorch#36]
Hello,
I train the source network, but I can't get 92.2% accuracy. Can you tell me how to get the accuracy? Thank you.
Below is the train results.
Train 240, loss: 1.343585, train acc: 0.991972, train avg acc: 0.987482
Test 240, loss: 1.442848, test acc: 0.921394, test avg acc: 0.894500
Train 241, loss: 1.344278, train acc: 0.991463, train avg acc: 0.987030
Test 241, loss: 1.440947, test acc: 0.920989, test avg acc: 0.894244
Train 242, loss: 1.343533, train acc: 0.991565, train avg acc: 0.987816
Test 242, loss: 1.445475, test acc: 0.921394, test avg acc: 0.896413
Train 243, loss: 1.343830, train acc: 0.990650, train avg acc: 0.986168
Test 243, loss: 1.442397, test acc: 0.925041, test avg acc: 0.899953
Train 244, loss: 1.342421, train acc: 0.991463, train avg acc: 0.987199
Test 244, loss: 1.445072, test acc: 0.921394, test avg acc: 0.887744
Train 245, loss: 1.342044, train acc: 0.991565, train avg acc: 0.986175
Test 245, loss: 1.439755, test acc: 0.920989, test avg acc: 0.890576
Train 246, loss: 1.344775, train acc: 0.990955, train avg acc: 0.986748
Test 246, loss: 1.443565, test acc: 0.923825, test avg acc: 0.901326
Train 247, loss: 1.342665, train acc: 0.992683, train avg acc: 0.989145
Test 247, loss: 1.442239, test acc: 0.924230, test avg acc: 0.895576
Train 248, loss: 1.341297, train acc: 0.992276, train avg acc: 0.988662
Test 248, loss: 1.441331, test acc: 0.923825, test avg acc: 0.892285
Train 249, loss: 1.339861, train acc: 0.992785, train avg acc: 0.988616
Test 249, loss: 1.440879, test acc: 0.923015, test avg acc: 0.894663

**Comment by antao97** [comment]
The testing script actually uses the best model tested on the testing set, not the last model.

**Comment by csgogo** [comment]
Thank you,
but the best result is in line 246 and test acc: 0.923825, test avg acc: 0.901326, is this normal?

**Comment by antao97** [comment]
I think it's normal. Maybe the batch size you use is less than mine.
